x/crosschain/keeper/msg_server_vote_outbound_tx.go

```diff
@@ -63,6 +63,16 @@ func (k msgServer) VoteOutbound(
 ) (*types.MsgVoteOutboundResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
+	// Check if TSS exists
+	// It is almost impossible to reach this point without a TSS,
+	// as the check for TSS was already done when creating the inbound, but we check anyway.
+	// We also expect the tss.Pubkey to be the same as the one in the outbound params,
```

Comment:
This is continegent on the fact
	// As we would have processed all CCTXs before migrating

I think it would be fine not to have that in place ,in case we change the migration process at some point